torb/trigger_staging_build/service.py

```diff
@@ -15,55 +16,98 @@ def handler(event, context):
     determined using beanstalk_utils.whodaman, which requires ElasticBeanstalk
     IAM permissions
     '''
-    # determine if we are deploying to fourfront-webprod or fourfront-webprod2
+    # determine if we are deploying to a production env
     event['source_env'] = bs.whodaman()
-    if event['source_env'] == 'fourfront-webprod':
-        event['dest_env'] = 'fourfront-webprod2'
-    elif event['source_env'] == 'fourfront-webprod2':
-        event['dest_env'] = 'fourfront-webprod'
-    else:
-        # event['source_env'] is unexpected. Bail
+    source_env = event['source_env']
+    dest_env = None
+    if is_fourfront_env(source_env) and is_stg_or_prd_env(source_env):
+        # TODO: Making this cgap-compatible means making an adjustment indicated later in this file. -kmp 10-Apr-2020
```

Comment:
Leaving this as a TODO is okay I'd say. We never used Torb for CGAP anyway.

Reply:
Yeah, that was my plan. Thanks for confirming.